src/layouts/container/_index.scss

```diff
@@ -7,3 +7,7 @@
   @include size.max-width(130rem);
   @include padding.inline([ 'bee', 'grizzly' ]);
 }
+
+.sl-container--full-width {
+  @include padding.inline('none');
```

Comment:
The container component in Sauce isn't really doing its job properly. Rather than adding a modifier to override this one it might be better to just apply the max-width and `margin 0 auto` to the specific thing you're styling in your application.

That, and also the fact that we need to be more careful about what we add to Sauce. If we find that we're continually having to override this style several times, then we should look at the component and what it's doing for us then.